modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements_controller.rb

```diff
@@ -7,6 +7,9 @@ class NoticeOfDisagreementsController < AppealsApi::V2::DecisionReviews::NoticeO
     include AppealsApi::OpenidAuth
     include AppealsApi::PdfDownloads
 
+    skip_before_action :new_notice_of_disagreement
+    skip_before_action :find_notice_of_disagreement
```

Comment:
As I was making these changes I tried to break the relationship between this controller and its parent a bit more by removing these `before_action`s that create or find NODs - these actions assign the NOD in question to an instance variable that other methods in the parent controller subsequently expect to be defined. This pattern seems pretty brittle so I replaced a few of the `render` actions with new revisions that expect the NOD to be passed in as an argument. This is a littler clearer to me as a reader.

Reply:
I saw that you made sure that any methods that used `@notice_of_disagreement` in the V1 controller were copied to the V0 controller. Looks good!